docs/user-documentation/objects.md

```diff
@@ -1,26 +1,65 @@
+### Object structure and properties
 
-todo: insert a diagram.
+Islandora 7 has the concept of an "object" that is based on the Fedora 3.x object model: an object has properties such as an owner, date created, date modified, and status, and each object contains a set of files (known as "datastreams") identified by datastream IDs, such as RELS-EXT, DC, MODS, and OBJ. The relationships between the object and its datastreams is highly constrained: datastreams cannot exist without a parent object, and each object can have only one DC datastream, one RELS-EXT datastream, and so on.
 
-## What is an Islandora object?
+In Islandora 8, we can also talk about "objects", but the underlying relationships between a collection of properties and its associated files differ substantially from those that make up an Islandora 7.x object. An object in Islandora 8 is a Drupal node, along with associated "media" (which is Drupal 8's name for files and their properties; media are described in detail [elsewhere](datastreams.md).) For example, an Islandora 8 object created by the Drupal user "admin" (user ID 1) on February 24, 2019, has the following properties:
 
-!!! todo
-    * It’s a node, with maybe one or more medias and files attached (see “Datastreams are Media” section)
-    * The node has metadata in its fields (See "Metadata are Fields" section.)
-    * It has a URI, and a JSON-LD view in Drupal
-    * It goes into Fedora. (See todo make "In Fedora" section)
-    * By Drupal default, all published nodes are public, so see todo make Access Control section.
-    * As a Drupal node, it’s available to Views and gets indexed in Solr.
-    * We suggest using a taxonomy term on the node to control elements of its display or editing, e.g. in Islandora Demo there is a taxonomy term that sets the type: Audio, Binary, Collection, Image, and Video
-    * How do I make an Islandora object? (Make a node, then add a Media)
+```
+userid: 1
+title: "I am an Islandora 8 object"
+created: 1550703004
+changed: 1550703512
+uuid: 02932f2c-e4c2-4b7e-95e1-4aceab78c638
+type: islandora_object
+status: 1
+```
 
+These are Drupal node properties, which all Drupal nodes have (although not all Drupal nodes have a type of "islandora_object", some have a type of "page" or "article" or "some_custom_content_type"). These node properties directly correspond to an Islandora 7.x object's properties:
 
-!!! tip "7.x migration note"
-    #### Where are my Solution Packs?
-    If you are familiar with Islandora 7.x, a solution pack is a bundle representing objects of a certain 'type', that are associated with metadata forms, an ingest workflow that allows specific filetypes, derivative settings based on the uploaded object, an optional content viewer, and the look and feel of an object's page including what metadata to display to the user.
+Islandora 7.x object properties | Islandora 8.x node/object properties
+------------ | -------------
+owner | userid
```

Comment:
Another s/userid/uid

Sorry i missed this one earlier

Reply:
Done.